src/consensus/aft/test/main.cpp

```diff
@@ -1040,366 +1042,4 @@ DOCTEST_TEST_CASE("Exceed append entries limit")
-      0);
-
-    DOCTEST_REQUIRE(
-      1 ==
-      dispatch_all(
-        nodes,
-        ((aft::ChannelStubProxy*)r2.channels.get())
-          ->sent_append_entries_response));
-  }
-
-  DOCTEST_INFO(
-    "Node 1 and Node 2 proceed to compact at idx 2, where Node 0 has "
-    "compacted for a previous term");
-  {
-    DOCTEST_REQUIRE(r1.replicate(kv::BatchVector{{2, second_entry, true}}, 2));
-    DOCTEST_REQUIRE(r1.ledger->ledger.size() == 2);
-    r1.periodic(ms(10));
-    DOCTEST_REQUIRE(
-      ((aft::ChannelStubProxy*)r1.channels.get())->sent_append_entries.size() ==
-      2);
-
-    // Nodes 0 and 2 receive append entries and respond
-    DOCTEST_REQUIRE(
-      2 ==
-      dispatch_all(
-        nodes,
-        ((aft::ChannelStubProxy*)r1.channels.get())->sent_append_entries));
-    DOCTEST_REQUIRE(
-      ((aft::ChannelStubProxy*)r1.channels.get())->sent_append_entries.size() ==
-      0);
-
-    DOCTEST_REQUIRE(
-      1 ==
-      dispatch_all(
-        nodes,
-        ((aft::ChannelStubProxy*)r2.channels.get())
-          ->sent_append_entries_response));
-
-    // Node 0 will not respond here since it received an append entries it can
-    // not process [prev_idex (1) < commit_idx (2)]
-    DOCTEST_REQUIRE(
-      ((aft::ChannelStubProxy*)r0.channels.get())
-        ->sent_append_entries_response.size() == 0);
-
-    DOCTEST_INFO("Another entry from Node 1 so that Node 2 can also compact");
-    DOCTEST_REQUIRE(r1.replicate(kv::BatchVector{{3, third_entry, true}}, 2));
-    DOCTEST_REQUIRE(r1.ledger->ledger.size() == 3);
-    r1.periodic(ms(10));
-    DOCTEST_REQUIRE(
-      ((aft::ChannelStubProxy*)r1.channels.get())->sent_append_entries.size() ==
-      2);
-
-    // Nodes 0 and 2 receive append entries
-    DOCTEST_REQUIRE(
-      2 ==
-      dispatch_all(
-        nodes,
-        ((aft::ChannelStubProxy*)r1.channels.get())->sent_append_entries));
-    DOCTEST_REQUIRE(
-      ((aft::ChannelStubProxy*)r1.channels.get())->sent_append_entries.size() ==
-      0);
-
-    // Node 0 will now have an ae response which will return false because
-    // its log for index 2 has the wrong term (ours: 1, theirs: 2)
-    DOCTEST_REQUIRE(
-      ((aft::ChannelStubProxy*)r0.channels.get())
-        ->sent_append_entries_response.size() == 1);
-    DOCTEST_REQUIRE(
-      1 ==
-      dispatch_all_and_DOCTEST_CHECK(
-        nodes,
-        ((aft::ChannelStubProxy*)r0.channels.get())
-          ->sent_append_entries_response,
-        [](const auto& msg) {
-          DOCTEST_REQUIRE(msg.last_log_idx == 2);
-          DOCTEST_REQUIRE(!msg.success);
-        }));
-
-    DOCTEST_REQUIRE(r1.ledger->ledger.size() == 3);
-    DOCTEST_REQUIRE(r2.ledger->ledger.size() == 3);
-
-    DOCTEST_CHECK(r1.get_term() == 2);
-    DOCTEST_CHECK(r1.get_commit_idx() == 2);
-    DOCTEST_CHECK(r1.get_last_idx() == 3);
-
-    DOCTEST_CHECK(r2.get_term() == 2);
-    DOCTEST_CHECK(r2.get_commit_idx() == 2);
-    DOCTEST_CHECK(r2.get_last_idx() == 3);
-  }
-}
+}
```

Comment:
Axed in favour of the end to end committable.py for now. The existing raft test scaffolding doesn't have a way to accurately replicate signatures at the moment.